code/modules/mob/living/carbon/human/species_types/lesserangel.dm

```diff
@@ -0,0 +1,160 @@
+    
+/datum/species/langel
+	name = "Lesser Angel"
+	id = "langel"
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS)
+	mutant_bodyparts = list("wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "Angel")
+	use_skintones = 1
+	no_equip = list(SLOT_BACK)
```

Comment:
This means they will NEVER be able to ware ANYTHING on their back

Reply:
That was already in the base Angel code, and it is confirmed that they can wear satchel

Reply:
Odd, noted, but odd